drivers/timer/nrf_rtc_timer.c

```diff
@@ -232,9 +232,14 @@ uint64_t z_nrf_rtc_timer_get_ticks(k_timeout_t t)
  * @param[in] chan A channel for which a new CC value is to be set.
  *
  * @param[in] req_cc Requested CC register value to be set.
+ *
```

Comment:
Missing documentation of `bool exact` parameter.

Reply:
added